internal/actions/ctl.go

```diff
@@ -375,18 +377,25 @@ func (a *ctlFn) Type() plugintypes.ActionType {
 	return plugintypes.ActionTypeNondisruptive
 }
 
-func parseCtl(data string) (ctlFunctionType, string, variables.RuleVariable, string, error) {
+func parseCtl(data string) (ctlFunctionType, string, variables.RuleVariable, string, *regexp.Regexp, error) {
 	action, ctlVal, ok := strings.Cut(data, "=")
 	if !ok {
-		return ctlUnknown, "", 0, "", errors.New("invalid syntax")
+		return ctlUnknown, "", 0, "", nil, errors.New("invalid syntax")
 	}
 	value, col, ok := strings.Cut(ctlVal, ";")
 	var colkey, colname string
 	if ok {
 		colname, colkey, _ = strings.Cut(col, ":")
 	}
 	collection, _ := variables.Parse(strings.TrimSpace(colname))
-	colkey = strings.ToLower(colkey)
+	var re *regexp.Regexp
+	if len(colkey) > 2 && colkey[0] == '/' && colkey[len(colkey)-1] == '/' {
```

Comment:
What happens here if the last `/` is escaped? Should not be treated as regex, right? E.g. `/user/` ✅ , but `/user\/` 💀 .

Reply:
indeed, have to fix it. We should have one unique function for that because we do this in multiple places